src/components/carousel/carousel.styles.ts

```diff
@@ -76,10 +76,8 @@ export default css`
     overflow-x: hidden;
   }
 
-  .carousel__slides--dragging {
-  }
-
-  :host([vertical]) ::slotted(sl-carousel-item) {
+  :host([vertical]) ::slotted(sl-carousel-item),
+  :host([vertical]) ::slotted(.carousel__item) {
```

Comment:
adding flexibility to content project any element and have these styles apply simply by adding the `.carousel__item` class.

I am a bit hesitant on this one, I feel like it would be better if the carousel element attaches this class for the consumer on the slotted children, rather than requiring the consumer to do it. On the other hand, the slottable is the domain of the consumer so it feels a bit "wrong" to do this as the component author.